flags/wagtail_hooks.py

```diff
@@ -20,7 +20,7 @@ def register_flag_admin_urls():
                 url(r'^$', views.select_site, name='select_site'),
                 url(r'^(\d+)/$', views.index, name='list'),
                 url(r'^(\d+)/save/$', views.save, name='save'),
-                url(r'^(\w+)/delete/$', views.delete, name='delete'),
+                url(r'^(.+)/delete/$', views.delete, name='delete'),
```

Comment:
Hmm.. I can't think of exactly how this might break, but this seems a bit overly generous to me. What about using the regex defined [here](https://github.com/cfpb/wagtail-flags/blob/master/flags/forms.py#L8) to make sure that only valid flag names are matched?